api/fields/workflow_fields.py

```diff
@@ -17,7 +16,7 @@ def format(self, value):
             return {
                 "id": value.id,
                 "name": value.name,
-                "value": encrypter.obfuscated_token(value.value),
+                "value": "*" * 20,
```

Comment:
Please change the method instead of changing the hard-code string here.

https://github.com/langgenius/dify/blob/ffba341258b6ec96301c10279754481eff0db5bb/api/core/helper/encrypter.py#L11